messages/lightning.preview.app.md

```diff
@@ -63,15 +67,17 @@ Preparing to download
 
 Downloading
 
-# certificate.attention
+# certificate.installation.notice
+
+A self-signed certificate needs to be installed on your device before you can use your device for previewing. This is a one-time setup only which you can skip if you have installed the certificate already.
+
+# certificate.installation.skip.message
 
-╔═══════════╗
-║ ATTENTION ║
-╚═══════════╝
+Do you want to skip this step
 
 # certificate.installation.description
 
-If you have not done so already, please install the self-signed certificate on your device before proceeding. The certificate file is located at
+Please install the self-signed certificate on your device before proceeding. The certificate file is located at
 
```

Comment:
@moorejacqueline As I mentioned in a [comment on your PR](https://github.com/salesforcecli/plugin-lightning-dev/pull/64#discussion_r1645232377), I updated the strings to address a UX feedback that we had received. The feedback was to first show a message to the user indicating the need for installing a certificate (one time action) and ask if they've done this already and want to skip or not. If they choose to not skip then show instructions on how to install.

Please review these strings and let me know if anything needs to change.

Reply:
Thanks for tagging me! I'll add a few small suggestions to the new text -- I think I can't do it in this thread.